osf/models/sanctions.py

```diff
@@ -576,6 +585,9 @@ class Retraction(EmailApprovableSanction):
     justification = models.CharField(max_length=2048, null=True, blank=True)
     date_retracted = NonNaiveDateTimeField(null=True, blank=True)
 
+    def _get_registration(self):
```

Comment:
I added `_get_registration` to the Retraction class and based the registration retrieval off of what I was seeing in other class methods. Should I go ahead and ensure that the other functions in the class use `_get_registration` where appropriate or does that move too far out of scope? @brianjgeiger

Reply:
Probably just make a ticket for that.